### DISCUSSION TITLE
Emissions Boundaries & Scope For SCI Equation

### DISCUSSION THREAD

**Discussion by will-iamalpine** [Green-Software-Foundation/sci#88] [General]
Ben Davy showed a compelling visualization for his formula that outlines scope. I'm borrowing from [his blog post ](https://medium.com/teads-engineering/evaluating-the-carbon-footprint-of-a-software-platform-hosted-in-the-cloud-e716e14e060c) to propose that we do something similar in our SCI for clarity: 

![image](https://user-images.githubusercontent.com/8934290/134542955-3fc86efd-9891-4930-b8c5-0af8c8e68f93.png)
![image](https://user-images.githubusercontent.com/8934290/134548083-2c302710-0b83-44b2-af53-2e8cac071a6f.png)

Here are the definitions of Scope 1, 2, and 3 according to the GHG Protocol:
Scope 1 emissions are direct emissions from owned or controlled sources.
Scope 2 emissions are indirect emissions from the generation of purchased energy.
Scope 3 emissions are all indirect emissions (not included in scope 2) that occur in the value chain of the reporting company, including both upstream and downstream emissions.

Will need to account for who is reporting this. e.g. who is reporting
“If outsourced to the cloud, IT emissions previously accounted for under the Greenhouse Gas Protocol Scope 1 — emissions that are directly linked to the activities of an organization from sources that it owns and control — and Scope 2 — emissions from the generation of purchased energy — move to Scope 3, referring to all other emissions.”

**Comment by atg-abhishek** [comment]
Perhaps related to #58 as well? Thoughts @jawache @Henry-WattTime based on that discussion? On the one hand I see the benefits of having something similar to GHG scopes but also feel that we wanted to move away from how the scopes as per the GHG don't really do justice to how software systems are built and are hence problematic?

**Reply by jawache** [reply]
I would suggest putting as much effort as we can to distance ourselves from the GHG protocol :) It's going to be an uphill battle convincing people we are not a competitor to the GHG (we will fail if we try to compete with GHG) and the standard reaction is to compare us to GHG already.

Our equation is pretty good and encompasses everything I think much more cleanly than GHG, `SCI=(E*I+M)/R`, most people don't understand Scope 1,2,3 and it's not so straightforward, in non-cloud use cases, like windows or mobile or open-source, the energy usage is scope 3 not scope 2 and they don't have PUE's! Much better to just call energy, energy.

**Reply by atg-abhishek** [reply]
100% on this - we need to make sure to explicitly also justify why we feel that the GHG doesn't do justice because as you point out @jawache people will definitely come asking that question
